data-table-row-detail.html

```diff
@@ -5,7 +5,6 @@
         padding: 0 24px 0 24px;
         display: flex;
         align-items: center;
-        height: 48px;
```

Comment:
avoiding the height property gives enough flexibility in sizing and styling of row details section.